components/common/BlobListLightbox.tsx

```diff
@@ -0,0 +1,89 @@
+import { useServerConfig } from '@/shell/ConfigurationContext'
+import { ToolsOzoneModerationDefs } from '@atproto/api'
+import { useCallback } from 'react'
+import Lightbox from 'yet-another-react-lightbox'
+import Captions from 'yet-another-react-lightbox/plugins/captions'
+import Thumbnails from 'yet-another-react-lightbox/plugins/thumbnails'
+
+export const useBlobUrl = ({ authorDid }: { authorDid: string }) => {
+  const { appview } = useServerConfig()
+  const cdnUrl = appview?.endsWith('.bsky.app')
+    ? 'https://cdn.bsky.app'
+    : appview
```

Comment:
Wouldn't it be better to make that part of the server config ?

Reply:
yep but im trying to reduce server side touch points. i have a ticket on trello to send it from server.